server/accounts.go

```diff
@@ -4215,6 +4278,16 @@ func (tr *transform) transformSubject(subject string) (string, error) {
 	return tr.transform(tts)
 }
 
+func (tr *transform) getHashBucket(key string, numBuckets int) string {
+	h := fnv.New32a()
+	_, err := h.Write([]byte(key))
+	if err != nil {
+		return fmt.Sprintf("error: %v", err)
```

Comment:
For this hash, Write() cannot return any error, so would simply call `h.Write()` here... but if it did return an error, I think then that getHashBucket() would have to return (string, error) because sounds like bad things would happen if we were to use "error: <some error>" as a bucket name :-)

Reply:
I checked the fnv source code and indeed it only ever will return nil as the err (I thought of passing that error back as the partition/bucket name as it's the only kind of signaling mechanism available that there was a problem with the hashing (though I didn't think a problem with the hash would ever happen since it's just a hash function :)). The simplification is implemented in https://github.com/nats-io/nats-server/pull/2890/commits/ceb47f0298443758546029c469dc29336c515fdb